### DISCUSSION TITLE
Welcome to Video-To-Midi-Converter Discussions!

### DISCUSSION THREAD

**Discussion by venividiviciuss** [venividiviciuss/Video-To-Midi-Converter#1] [Announcements]
# WELCOME

**Comment by stavicnic** [comment]
I tried to run this program and I get the following error:
_tkinter.TclError: bitmap "./favicon/midi.ico" not defined
What am I doing wrong?

**Comment by stavicnic** [comment]
I didn't do anything wrong, the problem was packaging the project, in the end I ran it but unfortunately it doesn't work.
